deps/npm/node_modules/request/node_modules/qs/Readme.md

```diff
@@ -0,0 +1,233 @@
+# qs
```

Comment:
This appears to be causing problems for me locally – I have `README.md` on disk, so `git am` fails on this patch. I'll look into this more tomorrow morning! @Fishrock123 mind taking a look to see if you're having trouble with this file as well?